app/Models/Traits/HasWorkdays.php

```diff
@@ -73,6 +73,7 @@ public function setOtherWorkdaysDescriptionAttribute(?string $value): void
     protected function sumTotalWorkdaysAmounts(array $collections): int
     {
         // Assume that the types are balanced, and just return the value from `gender`
```

Comment:
I was saying to go ahead and replace this old comment with the new one. Please remove this one.